frontend/src/main/components/Chat/ChatPanel.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Stack } from 'react-bootstrap'; // Assuming you're using react-bootstrap for the Stack component
+import ChatMessageCreate from 'main/components/Chat/ChatMessageCreate';
+import ChatDisplay from 'main/components/Chat/ChatDisplay';
+
+const ChatPanel = ({ commonsId, messages: messagesProp, userCommons: userCommonsProp }) => {
```

Comment:
Use original prop names. Otherwise it's additional mental overhead.

Reply:
The props are only for use in the storybook. I thought it was more readable when seeing "If the prop exists, use it, otherwise use the backend data as normal" rather than seeing "If the message exists, use it instead of replacing it with the backend" which should be the normal behavior. Kinda hard to explain but hope it makes sense.

Reply:
Appending the word "Prop" to the end of a prop doesn't clear the confusion in the way you're thinking. Instead of renaming props and end up with 2 names for the same prop, give them more meaningful names to represent they are for stories.